kratos/python/add_utilities_to_python.cpp

```diff
@@ -575,7 +575,9 @@ void AddUtilitiesToPython(pybind11::module& m)
 
     // Read materials utility
     py::class_<ReadMaterialsUtility, typename ReadMaterialsUtility::Pointer>(m, "ReadMaterialsUtility")
+    .def(py::init<Model&>())
```

Comment:
why do you introduce new constructors?

Reply:
A new constructor doesn't break code, adds new ways to do the same thing

Reply:
BTW, this doesn't change any important behaviour

Reply:
ah now I understood, you do the reading in a second step right?

Reply:
With this you construct the utility, then you can read (without the need of passing everything at once). This way can be used from c++ and create tests

Reply:
I am definitely NOT positive about adding this constructor.